Logger/FileLogger.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        public string FilePath { get; }
+
+        public FileLogger(string filePath)
+        {
+            FilePath = filePath;
+
+            if (!File.Exists(FilePath))
+            {
+                var file = File.Create(filePath);
+                file.Close();
+            }
+        }
+        public override void Log(LogLevel logLevel, string message)
+        {
+            StreamWriter streamWriter = File.AppendText(this.FilePath);
```

Comment:
Typically we avoid using `this.` unless it is explicitly needed. However the naming guidelines ensure that it is almost never needed.

```suggestion
            StreamWriter streamWriter = File.AppendText(FilePath);
```